### ISSUE TITLE
Allow multiedges on graph insertion

### ISSUE THREAD

**Issue by renatillas** [giacomocavalieri/graph#1]

**Comment by giacomocavalieri** [comment]
Hello! I'm not sure I understand what multi edges means, could you show me a code example of what that would look like?

**Comment by renatillas** [comment]
Yeah sorry for the short explanation. I meant to have multiple edges from node a to node b, each with it's own label.

**Comment by giacomocavalieri** [comment]
Ah now I see, there's no need to change the underlying graph data structure, you could use a `List(a)` for that and update the edges by adding labels to that list. To do that you'd need a `update_edge` function to change a single edge at a time and I don't think I haven't implemented that yet, do you think that would be a nice addition?
